types/metro-config/configTypes.d.ts

```diff
@@ -33,7 +33,7 @@ export type GetTransformOptions = (
     entryPoints: ReadonlyArray<string>,
     options: GetTransformOptionsOpts,
     getDependenciesOf: (filePath: string) => Promise<string[]>,
-) => Promise<ExtraTransformOptions>;
+) => Promise<Partial<ExtraTransformOptions>>;
```

Comment:
@huntie I think this needs to be fixed upstream as well. It's not how it is used in the template: https://github.com/facebook/react-native/blob/0a3c55562b5ba43b00c21ca4d7b2cba0c8eb6206/template/metro.config.js#LL10-L15C8

Reply:
Thanks, will update now.